zkg.meta

```diff
@@ -3,4 +3,5 @@ description = Adds cluster node's interface to logs.
 tags = log, logging, conn, add interface, add worker
 script_dir = scripts
 depends =
-	bro >=2.5
+	zeek >=2.5
+version = 1.0.1
```

Comment:
I haven't found the `version` field in the zkg docs. Am I missing something here?

Reply:
almost certainly me missing something, modeled after updates in the salesforce modules, reviewing the docs it appears this is all done with tags...